roles/ansible_icinga_business_process/tasks/configuration_checks_business_process.yml

```diff
@@ -1,19 +1,21 @@
 ---
 
-- assert:
+- name: check the configuration of the business process
+  assert:
     that:
       - businessprocess.title is defined
       - businessprocess.owner is defined
-      - businessprocess.menu == True or businessprocess.menu == False
```

Comment:
this could probably be shortened to `businessprocess.menu | bool`

Reply:
wouldn't this evaluate to `false` and fail the playbook when the setting is set to false?

Reply:
`| bool` checks if `businessprocess.menu` is a boolean, e.g. `true, false, yes, no`. So this should not make the playbook fail.

Reply:
just tried it. unfortunately it just converts the value to bool and setting a variable to false fails the playbook

example output of our test playbook:
```
TASK [ansible_icinga_business_process : check the configuration of the nodes] ****************************************************************************************
failed: [localhost] (item={'name': 'service-db-status', 'displayname': 'service DB Cluster', 'operator': '2 of', 'visible': False, 'checks': [..]}) => {
    "ansible_loop_var": "node",
    "assertion": "node.visible | bool",
    "changed": false,
    "evaluated_to": false,
    "msg": "Assertion failed",
    "node": {
        ...
        ],
        "displayname": "service DB Cluster",
        "name": "service-db-status",
        "operator": "2 of",
        "visible": false
    }
}
```